modules/standard/IO.chpl

```diff
@@ -1368,33 +1368,33 @@ pragma "no doc"
 extern const QIO_CH_BUFFERED:c_int;
 
 pragma "no doc"
-extern const QIO_METHOD_DEFAULT:c_int;
+private extern const QIO_METHOD_DEFAULT:c_int;
 pragma "no doc"
-extern const QIO_METHOD_READWRITE:c_int;
+private extern const QIO_METHOD_READWRITE:c_int;
 pragma "no doc"
-extern const QIO_METHOD_PREADPWRITE:c_int;
+private extern const QIO_METHOD_PREADPWRITE:c_int;
 pragma "no doc"
-extern const QIO_METHOD_FREADFWRITE:c_int;
+private extern const QIO_METHOD_FREADFWRITE:c_int;
 pragma "no doc"
-extern const QIO_METHOD_MMAP:c_int;
+private extern const QIO_METHOD_MMAP:c_int;
 pragma "no doc"
-extern const QIO_METHODMASK:c_int;
+private extern const QIO_METHODMASK:c_int;
 pragma "no doc"
-extern const QIO_HINT_RANDOM:c_int;
+private extern const QIO_HINT_RANDOM:c_int;
 pragma "no doc"
-extern const QIO_HINT_SEQUENTIAL:c_int;
+private extern const QIO_HINT_SEQUENTIAL:c_int;
 pragma "no doc"
-extern const QIO_HINT_LATENCY:c_int;
+private extern const QIO_HINT_LATENCY:c_int;
 pragma "no doc"
-extern const QIO_HINT_BANDWIDTH:c_int;
+private extern const QIO_HINT_BANDWIDTH:c_int;
 pragma "no doc"
-extern const QIO_HINT_CACHED:c_int;
+private extern const QIO_HINT_CACHED:c_int;
 pragma "no doc"
-extern const QIO_HINT_PARALLEL:c_int;
+private extern const QIO_HINT_PARALLEL:c_int;
 pragma "no doc"
-extern const QIO_HINT_DIRECT:c_int;
+private extern const QIO_HINT_DIRECT:c_int;
 pragma "no doc"
-extern const QIO_HINT_NOREUSE:c_int;
+private extern const QIO_HINT_NOREUSE:c_int;
```

Comment:
I think these should remain public (and uncodumented) for now. They're all features you can use, they just don't have reasonable names. (QIO_FDFLAG_ and QIO_METHOD_)